src/site/index.njk

```diff
@@ -5,7 +5,7 @@ pageClass: dark-theme theme-black
 {% extends 'layouts/page.njk' %}
 
 {% block sidebar %}
-  <h1>Hawksworx</h1>
+  <h1>Hawksworx!!!!!!!</h1>
```

Comment:
❗️

Reply:
Ha ha!
Demoing preview builds. This one might not get merged!
....but sharp eyes!